assets/src/setup/pages/save/style.css

```diff
@@ -0,0 +1,35 @@
+.done__preview-container {
+	border: 2px solid #e8e8e8;
+	border-radius: 8px;
+	padding: 37px;
+	text-align: center;
+}
+
+.done__preview-container .phone {
+	max-height: 610px;
+	padding-bottom: 2rem;
+}
+
+.done__preview-container .phone::before {
+	height: 9px;
+	width: 80px;
+}
+
+.done__preview-iframe {
+	height: 610px;
+	width: 100%;
```

Comment:
This is turning out to be extremely narrow, like 240px:

![image](https://user-images.githubusercontent.com/134745/86175827-d9eb4c00-bad8-11ea-86fd-b9a41525a9e6.png)

Therefore, I think the iframe needs to _not_ be floated to the right and instead appear _below_ the copy. The iframe should then have a `max-width:640px`.

Reply:
Or perhaps a `max-width:480px`. Whatever the most common screen resolution is for mobile phones.

Example:

![image](https://user-images.githubusercontent.com/134745/86176300-947b4e80-bad9-11ea-8137-31c20f6e6cf1.png)

cc @jwold

Reply:
I just set a fixed width of 400px on that column (approximately what it is on the design) and have it wrapping below a certain width).

<img width="1051" alt="Screen Shot 2020-06-30 at 4 29 49 PM" src="https://user-images.githubusercontent.com/9020968/86178978-04e09a80-baef-11ea-9d3a-3acb814de408.png">